router/batchrouter/batchrouter_test.go

```diff
@@ -149,36 +149,38 @@ var _ = Describe("BatchRouter", func() {
 	var c *testContext
 
 	BeforeEach(func() {
+		config.Reset()
 		router_utils.JobRetention = time.Duration(175200) * time.Hour // 20 Years(20*365*24)
 		c = &testContext{}
 		c.Setup()
 	})
 
 	AfterEach(func() {
+		config.Reset()
```

Comment:
Do we need to reset it both in `BeforeEach` and `AfterEach` ?

Reply:
it depends, does the previous/next test in the suite resets? who knows!